src/data/equiv/basic.lean

```diff
@@ -61,6 +61,18 @@ equiv.ext _ _ H
 ⟨e₂.to_fun ∘ e₁.to_fun, e₁.inv_fun ∘ e₂.inv_fun,
   e₂.left_inv.comp e₁.left_inv, e₂.right_inv.comp e₁.right_inv⟩
 
+@[simp]
+lemma to_fun_as_coe (e : α ≃ β) (a : α) : e.to_fun a = e a := rfl
+
+@[simp]
+lemma inv_fun_as_coe (e : α ≃ β) (b : β) : e.inv_fun b = e.symm b := rfl
+
+@[simp]
+lemma left_inv_apply (e : α ≃ β) (a : α) : e.symm (e a) = a := e.left_inv a
+
+@[simp]
+lemma right_inv_apply (e : α ≃ β) (b : β) : e (e.symm b) = b := e.right_inv b
```

Comment:
And this as `apply_symm_apply`.

Reply:
BTW, both of these issues are reported by `#lint`.

Reply:
Oops.

I really wish there was a way to make a "private" PR, that acted for all intents and purposes as a PR, except that other people couldn't see them. As CI gets better, I'm relying on it more to do my building and linting for me, and the PR interface is easier to use than tracking down the CI reports for individual branches, so I start PRs too early, wasting everyones' time... I guess I'll try to learn to deal with CI reports for branches.

Reply:
Maybe we can give "Draft PR" that semantics? In other words: "Please don't waste your time on reviewing this. I'll let you know when it's ready."